openpower/scripts/fixup-target-fonts

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Remove (unused) Liberation fonts
+rm -rf $TARGET_DIR/usr/share/fonts/liberation
```

Comment:
Why are they installed in the first place?

Reply:
fbterm dependency.. I'll try to sort this out on the upstream submission with a virtual package

Reply:
Turns out that this is harder than I thought, because the buildroot dependency system only allows one "provider" per virtual package, which would mean that the customer would have to choose between liberation and wqy-zenhei if we made those two providers for a 'base font'.. I don't think this will fly with the community, so since this is only required because of our size limitations on the bootkernel part, I think it's OK to allow this (if the _defconfig requires it)